pkgs/development/libraries/sord/default.nix

```diff
@@ -1,12 +1,16 @@
-{ lib, stdenv, fetchurl, pkg-config, python3, serd, pcre, wafHook }:
+{ lib, stdenv, fetchFromGitHub, pkg-config, python3, serd, pcre, wafHook }:
 
 stdenv.mkDerivation rec {
   pname = "sord";
-  version = "0.16.4";
+  version = "0.16.8-20210112";
```

Comment:
If this is not a tag on github please use unstable-XXXX-XX-XX with the date of the commit.

Reply:
Was this documented somewhere?

Reply:
I am pretty sure it is in the contributing guide somewhere.